seq2seq/third_party/models/modeling_outputs.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
Is this a brand new file you wrote entirely from scratch, as it appears to be? If so, since it isn't an __init__.py file, please move it out of third_party and correct the copyright notice.

If it isn't, can you link me to the original version of the file?

Reply:
Please find the original file here: https://github.com/huggingface/transformers/blob/v3.5.1/src/transformers/modeling_outputs.py 

I only modified 'BaseModelOutputWithPastAndCrossAttentions' and 'Seq2SeqLMOutput'